docs/_menu-item_template.html

```diff
@@ -0,0 +1 @@
+<li class="nav-item"><a href="{{fsdocs-menu-item-link}}" class="nav-link">{{fsdocs-menu-item-content}}</a></li>
```

Comment:
Don't add these files to the `docs` folder, again the website https://fsprojects.github.io/FSharp.Formatting does not use this. It only adds confusing that these files are here.